seqr/views/apis/locus_list_api.py

```diff
@@ -114,6 +112,9 @@ def add_project_locus_lists(request, project_guid):
     for locus_list in locus_lists:
         locus_list.projects.add(project)
         locus_list.save()
+    logger.info('Updated Project {}'.format(project_guid), extra={'user': request.user, 'db_update': {
+        'dbEntity': 'Project', 'entityId': project_guid, 'updateType': 'update', 'updateFields': ['locus_lists'],
+    }})
```

Comment:
Yea, its because the project is being added the the `LocusList` model, but we also want to log that that means the project itself changes, because there is a many-to-many relationship between them. I can at least use a helper method for generating the log message though so I will do that